certvalidator/validate.py

```diff
@@ -1447,7 +1454,10 @@ def verify_crl(cert, path, validation_context, use_deltas=True, cert_description
                 certificate_list
             ))
             continue
-        if moment > certificate_list['tbs_cert_list']['next_update'].native:
+
+        # according to RFC 5280 5.1.2.5 next_update is optional
+        if certificate_list['tbs_cert_list']['next_update'] and \
+           moment > certificate_list['tbs_cert_list']['next_update'].native:
```

Comment:
RFC 5280 says that a conforming CRL issuer MUST provide this field.

Did you run into this in the wild?

Perhaps this is allowed in `soft-fail` mode, but not `hard-fail` or `require`?